### PR TITLE
Update the documentation for the FIELD keyword

### PR DESCRIPTION
**Pull request by frode-aarstad**
**Issue**
Resolves #my_issue

**Approach**
_Short description of the approach_

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Review by pinkwah — APPROVED** [review]

**Comment by oyvindeide** [comment]
Looks good! Can do some updates in: `data_types.rst` also though, for example: 
``` rst
1. The `FIELD` implementation can behave both as a dynamic quantity, i.e.
   pressure and saturation, and static property like porosity. In ERT it is
   currently *only used* as a parameter, but that this *possible* dual usage
   exists in the code adds a significant complexity.
```

File: docs/reference/configuration/data_types.rst [hunk 1]
```diff
@@ -532,8 +532,7 @@ Pecularites of EnKF heritage include:
 
 1. The `FIELD` implementation can behave both as a dynamic quantity, i.e.
```
Comment (oyvindeide) [inline docs/reference/configuration/data_types.rst:533]: Also a reference to dynamic here.

Comment (frode-aarstad) [reply docs/reference/configuration/data_types.rst:533]: I guess that FIELD here refers to fields in general while the next sentence states what ERT supports. 
If we remove the dynamic reference here I guess it makes sense to remove the whole paragraph.

Comment (oyvindeide) [reply docs/reference/configuration/data_types.rst:533]: Yes, agree that makes sense :+1:

**Pull request merged**